### ISSUE TITLE
Android build breaking in Turtle 0.8.4

### ISSUE THREAD

**Issue by Cawllec** [expo/turtle#102]
In our CI build for an Expo test fixture we're getting some failures regarding validation of our config on the latest releases of Turtle:
```
  | platform: "android"
  | buildPhase: "setting up environment"
  | Jun 25 12:35:22 turtle[6] INFO:  Using manifest: {"android":{"package":"com.bugsnag.expo.testfixture"},"bundledAssets":["asset_3a2ba31570920eeb9b1d217cabe58315.ttf","asset_744ce60078c17d86006dd0edabcd59a7.ttf","asset_140c53a7643ea949007aa9a282153849.ttf","asset_6beba7e6834963f7f171d3bdd075c915.ttf","asset_b06871f281fee6b241d60582ae9369b9.ttf","asset_e20945d7c929279ef7a6f1db184a4470.ttf","asset_b2e0fc821c6886fb3940f85a3320003e.ttf","asset_d15c1216957060fac577af6151fb8cfe.ttf","asset_a37b0c01c0baf1888ca812cc0508f6e2.ttf","asset_73b8cff012825060b308d2162f31dbb2.ttf","asset_d2285965fe34b05465047401b8595dd0.ttf","asset_5cdf883b18a5651a29a4d1ef276d2457.ttf"],"extra":{"bugsnag":{"apiKey":"645470b8c7f62177e1a723e26c9a48d7"}},"icon":"./assets/icon.png","iconUrl":"https://d1wp6m56sqw74a.cloudfront.net/~assets/9e094c2652e91fbe16ffcfd5a42e3afa","ios":{"bundleIdentifier":"com.bugsnag.expo.testfixture","supportsTablet":true},"locales":{},"name":"Test Fixture","orientation":"default","platforms":["ios","android"],"privacy":"unlisted","sdkVersion":"32.0.0","slug":"test-fixture","splash":{"backgroundColor":"#ffffff","image":"./assets/splash.png","imageUrl":"https://d1wp6m56sqw74a.cloudfront.net/~assets/43ec0dcbe5a156bf9e650bb8c15e7af6","resizeMode":"contain"},"updates":{"fallbackToCacheTimeout":0},"version":"2.0.0","id":"@bugsnag-notifiers/test-fixture","revisionId":"2.0.0-r.3VyN74OJPB","publishedTime":"2019-06-25T12:34:02.657Z","commitTime":"2019-06-25T12:34:02.700Z","bundleUrl":"https://d1wp6m56sqw74a.cloudfront.net/%40bugsnag-notifiers%2Ftest-fixture%2F2.0.0%2Fd8ca94eec12369565d856adf271a6e36-32.0.0-android.js","releaseChannel":"dceab747-e65d-4020-862a-c047dce8bf1a","hostUri":"exp.host/@bugsnag-notifiers/test-fixture?release-channel=dceab747-e65d-4020-862a-c047dce8bf1a"}
  | platform: "android"
  | buildPhase: "reading manifest"
  | Jun 25 12:35:22 turtle[6] ERROR: Failed to build standalone app
  | err: ValidationError: child "config" fails because [child "buildType" fails because ["buildType" must be one of [debug, release]]]
  | at Object.exports.process (./node_modules/joi/lib/errors.js:196:19)
  | at internals.Object._validateWithOptions (./node_modules/joi/lib/types/any/index.js:675:31)
  | at internals.Object.validate (./node_modules/joi/lib/types/any/index.js:709:21)
  | at Object.sanitizeJob (./node_modules/turtle-cli/src/validator.ts:5:35)
  | at ./node_modules/turtle-cli/src/bin/utils/builder.ts:75:25
  | at <anonymous>
  | at process._tickDomainCallback (internal/process/next_tick.js:229:7)
  | platform: "android"
```
I'm not entirely sure where this is coming from, as I can't see `buildType` defined anywhere.
We've since pinned to an older version, but it would be good to update to the latest turtle release.

**Comment by wkozyra95** [comment]
Hi
Are you sure that this is 0.8.4? We added this feature in 0.8.5, but because of unrelated problems on ios it was removed from npm within few minutes. Downgrading to 0.8.4 should fix that(this is currently the latest version of turtle-cli).

**Comment by Cawllec** [comment]
Hmm, my logs indicate 0.8.4 was installed, but it may have been a cached docker layer.  I'll clear our caches just incase 0.8.5 is still around somewhere.

**Comment by wkozyra95** [comment]
I just released version 0.8.6 that has the same changes as removed 0.8.5 and fix for the ios issue I mentioned.
